api/capi/src/nntrainer.cpp

```diff
@@ -337,14 +336,24 @@ int ml_train_model_compile(ml_train_model_h model, ...) {
   return status;
 }
 
-int ml_train_model_run(ml_train_model_h model, ...) {
+int ml_train_model_compile_with_params(ml_train_model_h model,
+                                       const char *params) {
+  std::vector<std::string> param_list;
+
+  check_feature_state();
+  ML_TRAIN_VERIFY_VALID_HANDLE(model);
+
+  if (params)
+    param_list = split_params(params, '`');
```

Comment:
I wonder how it can handle like 
``` c++
ml_train_model_compile_with_params("loss=cross, batch_size=32" )
```

Reply:
We had an offline meeting about this. Luckily, nntrainer handles exceptions well.
so We need to supplement the error message about adding parameter and also docs with example.